contrib/storage-hive/core/src/main/java/org/apache/drill/exec/expr/fn/HiveFunctionRegistry.java

```diff
@@ -53,12 +54,14 @@
    * @param config
    */
   public HiveFunctionRegistry(DrillConfig config) {
-    Set<Class<? extends GenericUDF>> genericUDFClasses = PathScanner.scanForImplementations(GenericUDF.class, null);
+    // TODO: see if we can avoid this. We can't change the constructor right now.
+    ScanResult classpathScan = ClassPathScanner.fromPrescan(config);
```

Comment:
no because fromPrescan has a cache (it keeps track of the last scan and reuses it if we ask for the same thing), but that's far from ideal.